suite-common/formatters/src/tests/MockedFormatterProvider.tsx

```diff
@@ -14,6 +14,7 @@ export const MockedFormatterProvider = ({ children }: MockedFormatterProviderPro
 
     const formatters = getFormatters({
         locale: 'en',
+        coins: [],
```

Comment:
You will be probably need to add some fiat rates here. (we should have them in `@suite-common/test-utils` mocks)

Reply:
I don't see any rates here.